### PR TITLE
doc: document vstart options

### PR DESCRIPTION
**Pull request by zhongyimao**

### CODE REVIEW

File: doc/dev/dev_cluster_deployement.rst [hunk 1]
```diff
@@ -37,49 +65,69 @@ Options
...
     Use ceph-run as wrapper for mon/osd/mds.
 
-.. option:: --smallmds
+.. option:: --nolockdep
 
-    Configure mds with small limit cache size.
+    Disable lockdep
 
-.. option:: -x
+.. option:: -o config
```
Comment (tchaikov) [inline doc/dev/dev_cluster_deployement.rst]: `s/config/<config>/`

File: doc/dev/dev_cluster_deployement.rst [hunk 2]
```diff
@@ -37,49 +65,69 @@ Options
...
 
-    Enable Cephx (on by default).
+    Add *config* to all sections in the ceph configuration.
 
-.. option:: -X
+.. option:: --rgw_port
```
Comment (tchaikov) [inline doc/dev/dev_cluster_deployement.rst]: `--rgw_port <port>`

File: doc/dev/dev_cluster_deployement.rst [hunk 3]
```diff
@@ -37,49 +65,69 @@ Options
...
 
-    Disable Cephx.
+    Specify ceph rgw http listen port.
 
-.. option:: -d, --debug
+.. option:: --rgw_frontend
```
Comment (tchaikov) [inline doc/dev/dev_cluster_deployement.rst]: `--rgw_frontend <frontend>`

the frontend is `civetweb` by default.

File: doc/dev/dev_cluster_deployement.rst [hunk 4]
```diff
@@ -37,49 +65,69 @@ Options
...
+
+    Specify the rgw frontend configuration.
+
+.. option:: --rgw_compression
```
Comment (tchaikov) [inline doc/dev/dev_cluster_deployement.rst]: `--rgw_compression <compression_type>`

disabled by default.

**Comment by zhongyimao** [comment]
@tchaikov Updated. Thank you very much.

File: doc/dev/dev_cluster_deployement.rst [hunk 5]
```diff
@@ -37,49 +65,69 @@ Options
...
+.. option:: -x
 
-    Use memstore as the objectstore backend for osds
+    Enable Cephx (on by default).
 
-.. option:: --cache <pool>
+.. option:: -X
 
-    Set a cache-tier for the specified pool
+    Disable Cephx.
```
Comment (tchaikov) [inline doc/dev/dev_cluster_deployement.rst]: please use `CephX` or `cephx`, to be consistent with other parts of the document.

**Comment by tchaikov** [comment]
aside from the nit, lgtm

File: doc/dev/dev_cluster_deployement.rst (hunk 5 above)
Comment (zhongyimao) [reply doc/dev/dev_cluster_deployement.rst]: ok, I will replace Cephx with cephx. Thanks.

**Pull request merged**